lxc/auth.go

```diff
@@ -756,6 +764,120 @@ func (c *cmdIdentity) command() *cobra.Command {
 	return cmd
 }
 
+type cmdIdentityCreate struct {
+	global     *cmdGlobal
+	flagGroups []string
+}
+
+func (c *cmdIdentityCreate) command() *cobra.Command {
+	cmd := &cobra.Command{}
+	cmd.Use = usage("create", i18n.G("[<remote>:]<authentication_method>/<name> [<path to PEM encoded certificate>] [[--group <group_name>]]"))
+	cmd.Short = i18n.G("Create a TLS identity")
```

Comment:
This command is specifically for creating a TLS identity and yet the usage format above seems to indicate one can provide different authentication methods?

Reply:
Ah I had in my mind to not rule out creation of other identity types. I'll just change to `Create an identity`.

I'll look into the stdin changes that Gabriel has worked on and do something similar.

Reply:
I've implemented creating an identity from stdin. As in the cherry-picks from incus, stdin is modified with command line args. It is assumed that if no certificate is provided (either via stdin or as an argument) that the caller wants to get a token. In that case the token field is set to true.